src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/CJ4_FMC_ModSettingsPage.js

```diff
@@ -17,6 +17,11 @@ class CJ4_FMC_ModSettingsPageOne {
         else {
             this._lightMode = CJ4_FMC_ModSettingsPage.LIGHT_MODE.OFF;
         }
+
+        this._gpuSetting = SimVar.GetSimVarValue("EXTERNAL POWER ON", "Bool");

```

Comment:
will this give you 0/1 ? or should you use "number"?

Reply:
There's helper code for this case in the SimVar method to return an actual bool.